pocs/observatory.py

```diff
@@ -203,6 +204,15 @@ def remove_camera(self, cam_name):
         self.logger.debug('Removing {}'.format(cam_name))
         del self.cameras[cam_name]
 
+    def add_scheduler(self, scheduler):
+        assert isinstance(scheduler, BaseScheduler)
+        self.logger.debug('Adding scheduler')
```

Comment:
```suggestion
        self.logger.info('Adding scheduler')
```

Let's put this at info level.